modules/ocf_printhost/files/enforcer

```diff
@@ -164,6 +174,16 @@ def send_printer_mail(message, job, quo):
 
 def prehook(c, r, job):
     quo = quota.get_quota(c, job.user)
+
+    # A stop gap until we figure out why some file types aren't printing correctly
+    for file_type, prohibited in PROHIBITED_FILE_TYPES:
```

Comment:
Do we expect to have more "prohibited file types"? If not, this kind of abstraction isn't really necessary. Just have an if statement for gdocs.

Reply:
Abizer mentioned there may be more file types, I figured I'd leave it extendable just in case.

Reply:
it stands to reason that Google Docs is not the only thing that fails - does the same occur for printing straight from bCourses or Gmail or something else? this amount of abstraction is hardly enterprise-ready, I think it's safe to plan for a printing-unfriendly future with this. I'd call it `PROBLEMATIC_FILE_SOURCES` or something like that though.